main/ErrorExtender.js

```diff
@@ -6,6 +6,8 @@ const validator = Assert.validator;
 const hro = require('./helpers/HiddenReadOnly');
 const merge = require('./helpers/Merge');
 
+const CAUSED_BY_PREFIX = 'Caused by:';
```

Comment:
I'm not very aware of this lib, but this matching seems very flaky, e.g by text, isn't there an aggregated state before producing output so that we are not coupled to this type of implementation?

Reply:
We add this prefix to the stack trace in this library. Just moved to a variable.